BeagleBone_black/src/motor_control/main.cpp

```diff
@@ -90,42 +93,42 @@ void Main::setupMotors()
   */
 void Main::accelerateMotors()
 {
-  while (state.current_state == data::State::kAccelerating) {
+  while (state_.current_state == data::State::kAccelerating) {
     // Check for state machine critical failure flag
-    state = data.getStateMachineData();
-    if (state.critical_failure) {
+    state_ = data_.getStateMachineData();
+    if (state_.critical_failure) {
       this->stopMotors();
       goto exit_loop;
     }
 
     // Check for motors critial failure flag
-    motorFailure = motor->checkStatus();
-    if (motorFailure) {
+    motorFailure_ = motor_.checkStatus();
+    if (motorFailure_) {
       log_.INFO("MOTOR", "Motor State: Motor Failure\n");
-      MotorsRpm motors_rpm = motor->getSpeed();
-      motor_data = {
+      MotorsRpm motors_rpm = motor_.getSpeed();
+      motor_data_ = {
         data::MotorState::kCriticalFailure,
-        motors_rpm.rpm_FL,
-        motors_rpm.rpm_FR,
-        motors_rpm.rpm_BL,
-        motors_rpm.rpm_BR };
-      data.setMotorData(motor_data);
+        motors_rpm.rpm_FL_,
+        motors_rpm.rpm_FR_,
+        motors_rpm.rpm_BL_,
+        motors_rpm.rpm_BR_ };
+      data_.setMotorData(motor_data_);
       this->stopMotors();
```

Comment:
It looks to me like you stop the motors here but then you set them running again in the next section of code. What did I miss?

Reply:
Good spot, thanks